### DISCUSSION TITLE
0.98.3.1 for MC 1.16.* & 1.17.* & 1.18.* & 1.19.

### DISCUSSION THREAD

**Discussion by LlmDl** [TownyAdvanced/Towny#6000] [Pre-Release Discussions]
- Fix /t outpost generating a array index out of bounds.

----

<table align=center>
<tr><th colspan=5><h3>Towny Sponsors<br><h4>I want to give a big thank you to all of my sponsors:<br>These are the people that help to make Towny's support and development as active as it is,<br>and who supported me during this pre-release of Towny. </th></tr>
<tr><td>MrLogano</td><td>Momshroom</td><td>Cobrex1</td><td> KarlOfDuty </td><td>scatterspell</td></tr>
<tr><td>TheFlagCourier</td><td>gerardvanschip</td><td>DrprofLuigi</td><td>SmallSansSerif</td><td>TheNylox</td></tr>
<tr><td>trevor1097</td><td>benevolent2012</td><td>Scottlander100</td><td>EnvyEnfys</td><td>MidnightSugar</td></tr>
<tr><td>mzyax</td><td>cigaming</td><td>KeNickGong</td><td>MilesBHuff</td><td>pebblehost</td></tr>
<tr><td>Akiranya</td><td>BamBamTheEggman</td><td>NoikzGaming</td><td>TakeoRenka</td><td>LordHenryVonHenry</td></tr>
<tr><td>SolarCiv</td><td>Litning11</td><td>SkapMC</td><td>Jannies</td><td>Lemuridaee</td></tr>
<tr><td>ReEvadere</td><td>qckcat</td><td>roflmarv</td><td>beabuenodev</td><td>LoJoSho</td></tr>
<tr><td>benjamin1313</td><td>ThePontificate</td><td>AllieDragon</td><td>TegulaThePig</td><td>benlmyers</td></tr>
<tr><td>KjellWolf</td><td>Caleston</td><td>Dynamite99</td><td>TuckerAndrews3</td><td>darthpeti</td></tr>
<tr><td>PixelJames22</td><td>Cboxhero</td><td>WillysHub</td><td>ColbertoSwagerto</td><td>Agaloth</td></tr>
<tr><td>Mille12313</td><td>LaBoulangerie</td><td>MrSneakyGuy</td><td>Eblyss</td><td>BurgBoi</td></tr>
<tr><td>akup001</td><td>Intrabit71</td><td>VexfulNiko</td><td>MetaverseRPN</td><td>KawadaJP</td></tr>
<tr><td>finder1793</td><td>mattematti2</td><td>gladmountain</td><td>5zv</td><td>havencraftmc</td></tr>
<tr><td>Bloc-k</td><td>flare2121</td><td>EchoGravity</td><td>SW1D3RSK1</td><td>Peskdt</td></tr>
<tr><td>xSTORMYxSHM</td><td>bdagenais2</td><td>Dantexde</td><td>txtrapline</td><td>FauultySpell</td></tr>
<tr><td>RenoldHenold</td><td>TamaKaiNya</td><td>AnachroSoft</td><td>echterSupJ</td><td>Laughingspade</td></tr>
<tr><td>MightyMat</td><td>0xBitMC</td><td>LightNing002</td><td>Kamen9981</td><td>Valerie2308</td></tr>
<tr><td>Zee1029</td><td>ethsmith</td><td>Wextra</td><td>Sanictr</td><td>Downsurge</td></tr>
<tr><td>lukemango</td><td>ethanic17</td><td>Chibby9607</td><td>Zumodenaranja</td><td>RulaAshvure</td></tr>
<tr><td>Kelbbie</td><td>zhihan369</td><td>HerbDashwell</td><td>KaiserSergio</td><td colspan=1>and 14 private sponsors.</td></tr>
<tr><td colspan=5 align=center>
<h3><a href=https://github.com/sponsors/LlmDl>If you want to support the developer, become a sponsor.</a></h3>
(It's just like Patreon but instead 100% of your support goes to the developer.)
<h3>Plus there are <a href="https://github.com/TownyAdvanced/Towny/wiki/Sponsor-Perks">Sponsor Perks!</a></h3>
</td></tr>
</table>
<table align=center>
<th colspan=3> Important Links </th>
<tr align=center><td><a title="How Towny Works" href="https://github.com/TownyAdvanced/Towny/wiki/How-Towny-Works">How Towny 
Works</a></td><td><a title="Install Guide" href="https://github.com/TownyAdvanced/Towny/wiki/Installation">Towny Install Guide</a></td><td><a title="Updating Towny" href="https://github.com/TownyAdvanced/Towny/wiki/Updating-Towny">Towny Update Guide</a></td>
</tr>
<tr align=center>
<td><a title="Towny Changelog" href="https://raw.githubusercontent.com/TownyAdvanced/Towny/master/resources/ChangeLog.txt" target="_blank" rel="noopener">Complete Changelog</a> </td><td><a title="Default Config Files" href="https://github.com/TownyAdvanced/Towny/wiki/Config-Files">Default Config Files</a></td><td><a title="Commands/Permissions/Placeholders" href="https://github.com/TownyAdvanced/Towny/wiki/Reference">Commands/Permissions<br>Placeholders</a></td>
</tr>
</table>

### ⏬ Download available as a .jar file in the Assets section below:

<hr /><em>This discussion was created from the release <a href='https://github.com/TownyAdvanced/Towny/releases/tag/0.98.3.1'>0.98.3.1 for MC 1.16.* & 1.17.* & 1.18.* & 1.19.</a>.</em>

**Comment by 2869949486** [comment]
its so cool
